### DISCUSSION TITLE
i created a video on how to do it. in spanish

### DISCUSSION THREAD

**Discussion by tecdisol** [tmijail/Odoo-Power-BI-Connector#4] [Show and tell]
really thanks for you development. i was very helpfull

Conectar ODOO como (Origen de Datos) en (Microsoft Power BI)
[https://youtu.be/mLrexWGPeUE](https://youtu.be/mLrexWGPeUE)

**Comment by tmijail** [comment]
Espectacular! Mañana lo miro.
